src/main/java/redis/clients/jedis/BasicCommands.java

```diff
@@ -41,4 +41,6 @@
   String configResetStat();
 
   Long waitReplicas(int replicas, long timeout);
+
+  String readonly();
```

Comment:
It's really of use for standalone connections to a redis cluster. If you connect to a slave redis custer node with a normal Jedis connection(that has a copy of the slot) and perform a query, it will still redirect you. If you set readonly, it will serve the answer it has. I'm not sure how this would apply to cluster, since Jedis cluster will always send you to the master for that slot.